hedera-node/src/main/java/com/hedera/services/state/expiry/ExpiryManager.java

```diff
@@ -95,7 +95,8 @@ public void restartEntitiesTracking() {
 
 		var expiries = new ArrayList<Map.Entry<Pair<Long, Consumer<EntityId>>, Long>>();
 		schedules.forEach((id, schedule) -> {
-			var pair = new Pair<Long, Consumer<EntityId>>(id.getNum(), scheduleStore::expire);
+			Consumer<EntityId> consumer = scheduleStore::expire;
+			var pair = Pair.of(id.getNum(), consumer);
```

Comment:
Couple of thoughts:

1. In terms of minimizing change, it might be better to not create line 98, but just call scheduleStore::expire inline as before?
2. If you're going to use line 98, is it necessary to declare "Consumer<EntityId>" on line 98? The line below is using "var". We should be consistent.
3. I prefer "final var" myself.

Reply:
`Pair.of` parameters cannot be a functional reference. 
I tried using `var` but method reference requires an explicit target type.